jenkins-scripts/dsl/ignition_collection.dsl

```diff
@@ -189,7 +189,85 @@ ignition_collection_jobs =
         'sdformat-ci-sdformat9-homebrew-amd64',
         'sdformat-ci-sdformat9-windows7-amd64',
         'sdformat-install-sdformat9_pkg-bionic-amd64'
-  ]
+  ],
+  'dome' : [
+        // 'ign_gazebo-ign-3-win',
+        // 'ign_gui-ign-3-win',
+        // 'ign_physics-ign-2-win',
+        // 'ign_rendering-ign-3-win',
+        // 'ign_sensors-ign-3-win',
+        //'ignition_dome-ci-default-homebrew-amd64',
+        //'ignition_dome-install-pkg-bionic-amd64',
+        // 'ignition_dome-install_bottle-homebrew-amd64',
+        'ignition_cmake-ci-ign-cmake2-bionic-amd64',
+        'ignition_cmake-ci-ign-cmake2-homebrew-amd64',
+        'ignition_cmake-ci-ign-cmake2-windows7-amd64',
+        'ignition_cmake2-install-pkg-bionic-amd64',
+        'ignition_cmake2-install_bottle-homebrew-amd64',
+        'ignition_common-ci-ign-common3-bionic-amd64',
+        'ignition_common-ci-ign-common3-homebrew-amd64',
+        'ignition_common-ci-ign-common3-windows7-amd64',
+        'ignition_common3-install-pkg-bionic-amd64',
+        'ignition_common3-install_bottle-homebrew-amd64',
+        // 'ignition_fuel-tools-ci-ign-fuel-tools5-bionic-amd64',
+        // 'ignition_fuel-tools-ci-ign-fuel-tools5-homebrew-amd64',
+        // 'ignition_fuel-tools-ci-ign-fuel-tools5-windows7-amd64',
+        // 'ignition_fuel-tools5-install-pkg-bionic-amd64',
+        // 'ignition_fuel-tools5-install_bottle-homebrew-amd64',
+        // 'ignition_gazebo-ci-ign-gazebo4-bionic-amd64',
+        // 'ignition_gazebo-ci-ign-gazebo4-homebrew-amd64',
```

Comment:
I think we can use the master branch CI jobs for now until the release branches are available

Reply:
7209c43, running collection again.